rules/windows/credential_access_lsass_openprocess_api.toml

```diff
@@ -127,12 +127,20 @@ api where host.os.type == "windows" and
         "?:\\ProgramData\\Microsoft\\Windows Defender\\Platform\\*\\MsMpEng.exe",
         "?:\\Program Files (x86)\\Asiainfo Security\\OfficeScan Client\\NTRTScan.exe",
         "?:\\Program Files (x86)\\Blackpoint\\SnapAgent\\SnapAgent.exe",
+        "?:\\Program Files (x86)\\CheckPoint\\Endpoint Security\\EFR\\EFRService.exe",
+        "?:\\Program Files (x86)\\CyberCNSAgent\\osqueryi.exe",
```

Comment:
Would it be better to eventually start adding exclusions from signature subject name since we have some grouped executables here?

Reply:
Microsoft Defender for endpoint doesn't have process signer information 😅